controller-server/src/main/java/com/yahoo/vespa/hosted/controller/notification/NotificationContent.java

```diff
@@ -0,0 +1,40 @@
+package com.yahoo.vespa.hosted.controller.notification;
+
+import java.net.URI;
+import java.util.Objects;
+
+/**
+ * Contains formatted text that can be displayed to a user to give extra information and pointers for a given
+ * Notification.
+ *
+ * @author enygaard
+ */
+public class NotificationContent {
+    private final String prettyType, messagePrefix;
+    private final URI uri;
+
+    private final Notification notification;
+
+    public NotificationContent(Notification notification, String prettyType, String messagePrefix, URI uri) {
+        this.notification = Objects.requireNonNull(notification);
+        this.prettyType = Objects.requireNonNull(prettyType);
+        this.messagePrefix = Objects.requireNonNull(messagePrefix);
+        this.uri = Objects.requireNonNull(uri);
+    }
+
+    public String getPrettyType() {
+        return prettyType;
+    }
+
+    public String getMessagePrefix() {
+        return messagePrefix;
+    }
+
+    public URI getURI() {
+        return uri;
+    }
+
+    public Notification getNotification() {
+        return notification;
+    }
```

Comment:
For the most part we don't prefix getters with `get`, to avoid "get-get-get-get-get-get" when accessing nested members.

Reply:
right-o